policy-link-mapping.txt

```diff
@@ -1,3 +1,4 @@
+./BSD%203-Clause%20License.md=bsd-license
```

Comment:
```suggestion
./BSD%203-Clause%20License.md=https://github.com/whatwg/sg/blob/master/BSD%203-Clause%20License.md
```

Reply:
Is this to fix the Markdown version, or the generated HTML? I checked the generated HTML and thought it was correct, did I actually mess it up?

Reply:
Oh, the idea is to publish this at `https://whatwg.org/bsd-license`?